runtime/chxvm_defs.py

```diff
@@ -216,7 +216,7 @@ def sigil(typ):
     ('Elu', [Array('x'), Float('alpha')], ['y']),
     ('Floor', [Array('x')], ['y']),
     ('Ceil', [Array('x')], ['y']),
-    ('Shape', [Array('data')], ['shape']),
+    ('Shape', [Array('data')], ['shape'], False),
```

Comment:
This is fine for now. However, I think we should introduce `Shape` type to this IDL-ish script. The Shape op should look like `('Shape', [Array('data')], [Shape('shape')])` and Reshape will look like `('Reshape', [Array('data'), Shape('shape')], ['reshaped'])`.